src/edu/csus/ecs/pc2/core/imports/clics/EventFeed.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.core.imports.clics;
+
+import java.util.HashMap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.model.JSONObjectMapper;
+
+/**
+ * 
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ */
+
+//@JsonIgnoreProperties(ignoreUnknown = true)
+public class EventFeed {
```

Comment:
The other classes in this package put "CLICS" as the first thing in the class name, which I think is a good thing since it helps make it clear that the class is defining something out of the (a) CLICS specification.  I think this class should follow the same guideline:  the class name should be something like "CLICSEventFeed".

Reply:
refactoring  EventFeed to CLICSEventFeedEvent

Reply:
pushed refactor